app/src/main/java/cm/aptoide/pt/view/DeepLinkManager.java

```diff
@@ -279,6 +322,39 @@ private void scheduleDownloadsDeepLink(Uri uri) {
     }
   }
 
+  private void myStoreDeepLink() {
+    subscriptions.add(accountManager.accountStatus()
+        .first()
+        .map(account -> {
+          if (account.isLoggedIn()) {
+            return account;
+          } else {
+            return null;
+          }
+        })
+        .subscribe(navigation -> {
+          if (navigation != null) {
+            appShortcutsAnalytics.shortcutNavigation(ShortcutDestinations.MY_STORE);
+            storeAnalytics.sendStoreOpenEvent(APPS_SHORTCUTS, navigation.getStore()
+                .getName(), false);
+            String theme = navigation.getStore()
+                .getTheme();
+            Logger.v(TAG, theme);
```

Comment:
Is it important to have the Logger.v(TAG, theme)? if not, you can remove the navigation.getStore.getTheme() to the navigateTo method. Up to you, you know the importance of the log better than I do. imo I don't think it's necessary, but you decide.

Reply:
Fixed